functions/export-to-csv/2.0/function.json

```diff
@@ -0,0 +1,96 @@
+{
+  "description": "Create a CSV from a collection of Records",
+  "label": "Export to CSV",
+  "category": "Utilities",
+  "icon": "CreateIcon",
```

Comment:
I think we should use a different icon for the export function (team Actions will have to add the icons to the IDE and json schema first)